### PR TITLE
GH-13290 Add unit tests to the ChannelGroupDisableCmd mmctl command

### PR DESCRIPTION
**Pull request by lurcio**
#### Summary
Add unit tests to the "ChannelGroupDisableCmd" mmctl command.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/13290

### CODE REVIEW

File: commands/group_test.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		err := channelGroupDisableCmdF(s.client, &cobra.Command{}, []string{channelArg})
+		s.Require().NotNil(err)
+		s.Require().Len(printer.GetLines(), 0)
+		s.Require().Len(printer.GetErrorLines(), 0)
+		s.EqualError(err, "Unable to find channel '"+channelArg+"'")
+	})
+}
```
Comment (jespino) [inline commands/group_test.go]: Can we add here some tests for the cases where the API calls fail?

**Comment by jasonblais** [comment]
Hey @lurcio, let us know if you have any questions about the feedback above? We'd be happy to clarify :)

**Comment by lurcio** [comment]
@jasonblais Thanks for the prompt. I'll add the requested changes in the next day or two.

File: commands/group_test.go (hunk 1 above)
Comment (lurcio) [reply commands/group_test.go]: @jespino Sure! I have added more tests now

**Pull request merged**